orchestrator-salt/src/main/resources/salt/salt/post-recipes/scripts/createuserhome.sh

```diff
@@ -71,25 +71,47 @@ PATH_PREFIX="/user"
 
 export KRB5CCNAME=/tmp/krb5cc_cloudbreak_$EUID
 
+
+klist -s
+if [[ $? -ne 0 ]]; then
+  kinit_as_hdfs
+fi
+
 WEBHDFS_HTTP_POLICY=$(hdfs getconf -confkey dfs.http.policy)
 
-if [ "$WEBHDFS_HTTP_POLICY" == "HTTP_ONLY" ]; then
-  NAMENODE_HTTP_ADDRESS=$(hdfs getconf -confkey dfs.namenode.http-address)
-  WEBHDFS_URL=http://$NAMENODE_HTTP_ADDRESS/webhdfs/v1
+hdfs haadmin -getAllServiceState
```

Comment:
nit: you can put this in the `if` statement itself to check status code instead of a separate line if you wanted.

Reply:
I fixed this, could you check and resolv if you find it ok?

Reply:
Looks good - didn't run it - but surface looks good.